lib/eventum/class.link_filter.php

```diff
@@ -263,7 +263,9 @@ public static function markdownFormat($text)
             $parser->enableNewlines = true;
         }
 
-        $text = $parser->parseParagraph($text);
+        $text = $parser->parse($text);
+        // strip paragraph, confuses single line areas
+        $text = preg_replace("{^<p>(.+)</p>\n$}", '$1', $text);
```

Comment:
note, that this will allow creating markdown that has `</p><p>` inside which results in invalid HTML. You should probably have two different `markdownFormat()` methods, one for "single line" context and another for longer texts.